src/main/resources/application.yaml

```diff
@@ -15,3 +15,5 @@ management:
 spring:
   application:
     name: ucmc-service
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
this doesn't feel right, what's the reason behind this change?

Reply:
Preview was complaining about this bean. `core-case-data-api.FeignClientSpecification`. We have to do similar change in claim-store since sprintboot version 2.1. Do we have alternate solution? Would be happy to adopt that.

```  
OverrideException: Invalid bean definition with name 'core-case-data-api.FeignClientSpecification' defined in null: Cannot register bean definition [Generic bean: class [org.springframework.cloud.openfeign.FeignClientSpecification]; scope=; abstract=false; lazyInit=null; autowireMode=0; dependencyCheck=0; autowireCandidate=true; primary=false; factoryBeanName=null; factoryMethodName=null; initMethodName=null; destroyMethodName=null] for bean 'core-case-data-api.FeignClientSpecification': There is already [Generic bean: class [org.springframework.cloud.openfeign.FeignClientSpecification]; scope=; abstract=false; lazyInit=null; autowireMode=0; dependencyCheck=0; autowireCandidate=true; primary=false; factoryBeanName=null; factoryMethodName=null; initMethodName=null; destroyMethodName=null] bound.
2020-07-07T08:39:39.983 INFO  [main] o.s.b.a.l.ConditionEvaluationReportLoggingListener
```

Reply:
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes#bean-overriding

Reply:
someone's been copy and pasting 😛 
https://github.com/hmcts/ccd-client/blob/master/src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataApi.java#L27
https://github.com/hmcts/ccd-client/blob/master/src/main/java/uk/gov/hmcts/reform/ccd/client/CaseEventsApi.java#L17

shall we fix this in ccd-client rather than accepting the error?

Reply:
Well spotted @swalker125, that's why we should avoid bean overriding, it usually hides the real problem. In the meanwhile we could go for previous `4.6.4` version of ccd-client

Reply:
https://github.com/hmcts/ccd-client/pull/60